components/Footer.js

```diff
@@ -1,40 +1,86 @@
 import Link from 'next/link';
 import React from 'react';
+import styles from '../styles/components/Footer.module.scss';
 import SocialMedia from './SocialMedia';
 
+// TODO: add GM?
+const footerACMLinks = [
+	{ title: 'About', path: '/about' },
+	{ title: 'Committees', path: '/committees' },
+	{ title: 'Events', path: '/events' },
+	{ title: 'Sponsors', path: '/sponsors' },
+	{ title: 'Tech Gala', path: '/techgala' },
+	{ title: 'Membership Portal', path: 'https://members.uclaacm.com', ext: true },
+];
+
+// Do we want to prepend ACM here?
+const footerCommitteeLinks = [
+	{ title: 'AI', path: 'https://uclaacmai.github.io/', ext: true },
+	{ title: 'Cyber', path: 'https://acmcyber.com', ext: true },
+	{ title: 'Design', path: '/committees#design' },
+	{ title: 'Hack', path: 'https://hack.uclaacm.com', ext: true },
+	{ title: 'ICPC', path: 'https://icpc.uclaacm.com', ext: true },
+	{ title: 'Studio', path: 'http://localhost:3000/committees#studio' },
+	{ title: 'Teach LA', path: 'https://teachla.uclaacm.com', ext: true },
+	{ title: 'W', path: '/committees#w' },
+];
+
+function FooterLinkElement({ title, path, ext }){
+	return (
+		<li>
+			<Link href={path}>
+				<a className={styles['link-footer']} target={ext ? 'target': ''} rel={ext ? 'noopener noreferrer' : ''}>{title}</a>
```

Comment:
I've only seen `target="_blank"` before, but verified that this works locally so lgtm

Reply:
Ooh great catch, it should be `target="_blank"` - my bad!